src/graphql/event/ldap.rs

```diff
@@ -12,64 +12,76 @@ pub(super) struct LdapBruteForce {
 
 #[Object]
 impl LdapBruteForce {
+    /// Start Time
     async fn time(&self) -> DateTime<Utc> {
         self.inner.time
     }
 
+    /// Source IP (Address)
     async fn src_addr(&self) -> String {
         self.inner.src_addr.to_string()
     }
 
+    /// Source Country
     /// The two-letter country code of the source IP address. `"XX"` if the
     /// location of the address is not known, and `"ZZ"` if the location
     /// database is unavailable.
     async fn src_country(&self, ctx: &Context<'_>) -> String {
         country_code(ctx, self.inner.src_addr)
     }
 
+    /// Source Customer
     async fn src_customer(&self, ctx: &Context<'_>) -> Result<Option<Customer>> {
         let store = crate::graphql::get_store(ctx).await?;
         let map = store.customer_map();
         find_ip_customer(&map, self.inner.src_addr)
     }
 
+    /// Source Network
     async fn src_network(&self, ctx: &Context<'_>) -> Result<Option<Network>> {
         let store = crate::graphql::get_store(ctx).await?;
         let map = store.network_map();
         find_ip_network(&map, self.inner.src_addr)
     }
 
+    /// Destination IP (Address)
     async fn dst_addr(&self) -> String {
         self.inner.dst_addr.to_string()
     }
 
+    /// Destination Country
     /// The two-letter country code of the destination IP address. `"XX"` if the
     /// location of the address is not known, and `"ZZ"` if the location
     /// database is unavailable.
     async fn dst_country(&self, ctx: &Context<'_>) -> String {
         country_code(ctx, self.inner.dst_addr)
     }
 
+    /// Destination Customer
     async fn dst_customer(&self, ctx: &Context<'_>) -> Result<Option<Customer>> {
         let store = crate::graphql::get_store(ctx).await?;
         let map = store.customer_map();
         find_ip_customer(&map, self.inner.dst_addr)
     }
 
+    /// Destination Network
     async fn dst_network(&self, ctx: &Context<'_>) -> Result<Option<Network>> {
         let store = crate::graphql::get_store(ctx).await?;
         let map = store.network_map();
         find_ip_network(&map, self.inner.dst_addr)
     }
 
+    /// Destination Port (Number)
     async fn dst_port(&self) -> u16 {
         self.inner.dst_port
     }
 
+    /// Protocol Number
     async fn proto(&self) -> u8 {
         self.inner.proto
     }
 
+    /// User Password List
```

Comment:
This function’s description does not appear in the Notion page. Could you please confirm if this description is correct?